android-components/components/support/utils/src/test/java/mozilla/components/support/utils/WebURLFinderTest.kt

```diff
@@ -59,6 +59,11 @@ class WebURLFinderTest {
         )
         assertEquals("http://ß.de/", find("http://ß.de/ çnn.çơḿ"))
         assertEquals("htt-p://ß.de/", find("çnn.çơḿ htt-p://ß.de/"))
+        assertEquals(
+            "http://[2001:db8::1.2.3.4]:8080/inner#anchor&arg=1",
+            find("test.com http://[2001:db8::1.2.3.4]:8080/inner#anchor&arg=1"),
+        )
+        assertEquals("http://[::]", find("test.com http://[::]"))
```

Comment:
Can we also add some more similar test coverage from UrlStringUtils into WebUrlFinderTest?

Reply:
Added `isUrlLikeEmulated()`, copied all the test cases, and marked cases that differ with `INVERT()`.